apps/prairielearn/src/pages/instructorAssessmentOverrides/instructorAssessmentOverrides.js

```diff
@@ -0,0 +1,78 @@
+const error = require('@prairielearn/error');
+const express = require('express');
+const asyncHandler = require('express-async-handler');
+const router = express.Router();
+const { loadSqlEquiv } = require('@prairielearn/postgres');
+const sqldb = require('@prairielearn/postgres');
+const sql = loadSqlEquiv(__filename);
+
+router.get(
+  '/',
+  asyncHandler(async (req, res) => {
+    const params = {
+      assessment_id: res.locals.assessment.id,
+      current_student_uid: res.locals.user.uid,
+      current_assessment_title: res.locals.assessment.title,
```

Comment:
I believe these aren't used in the query, is that so? If yes, please remove them.